FirebaseAI/Sources/AILog.swift

```diff
@@ -75,10 +75,10 @@ enum VertexLog {
   /// Subsystem that should be used for all Loggers.
   static let subsystem = "com.google.firebase"
 
-  /// Log identifier for the Vertex AI SDK.
+  /// Log identifier for the AI SDK.
   ///
   /// > Note: This corresponds to the `category` in `OSLog`.
-  static let service = "[FirebaseVertexAI]"
+  static let service = "[FirebaseAI]"
```

Comment:
I believe this is the only externally visible part of this change. I noticed the old service while running the quickstart

Reply:
Agreed. Thanks for catching!